src/pages/research-example.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Layout from "../components/common/layout";
+import KeyStats from "../components/key-stats/key-stats";
+import Hero from "../components/hero/hero";
+
+const IndexPage = () => {
```

Comment:
might be worth keeping a TODO here to remind us that this page needs to be removed

Reply:
done - removed the pills page too since we can have both demoed in one